code/__HELPERS/_lists.dm

```diff
@@ -331,6 +331,10 @@
 /proc/sortList(list/L, cmp=/proc/cmp_text_asc)
 	return sortTim(L.Copy(), cmp)
 
+//sortList but with assoc lists
+/proc/sortAssocList(list/L, cmp=/proc/cmp_text_asc)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Reply:
Should I change the other sort procs too for consistency?

Reply:
Just keep it here for now, atomization and all.

Reply:
To be fair I'm not even using it anymore actually